testkit/testdata/testdata.go

```diff
@@ -132,24 +150,6 @@ func (td *TestData) GetTestCases(t *testing.T, in interface{}, out interface{})
 	td.output[casesIdx].decodedOut = out
 }
 
-// OnRecord execute the function to update result.
-func (td *TestData) OnRecord(updateFunc func()) {
-	if record {
-		updateFunc()
-	}
-}
-
```

Comment:
seems you changed OnRecord from method of TestData to a function, how can later codes like `testdata.OnRecord()` work?

Reply:
If you take a look at the body, `OnRecord` is a function more than a method. Later codes `testdata.OnRecord` works because `testdata` there is the package name.

Reply:
thanks. get it. i thougt testdata is a var then can't finger out why it works..